cadl-tests/generate.sh

```diff
@@ -41,17 +41,9 @@ rm -rf ./existingcode
 # run cadl ranch tests sources
 cp -rf node_modules/@azure-tools/cadl-ranch-specs/http .
 
-export NAMESPACE=com.cadl.testserver.servicedriven1
-generate ./http/resiliency/srv-driven-1/main.cadl
-
-export NAMESPACE=com.cadl.testserver.servicedriven2
-generate ./http/resiliency/srv-driven-2/main.cadl
-
 unset NAMESPACE
```

Comment:
Do we have other place setting the env? If not, remove this as well.

Reply:
no other places have NAMESPACE env var.